wagtail/images/tests/test_admin_views.py

```diff
@@ -343,7 +367,7 @@ def test_num_queries(self):
         self.get()
 
         # Initial number of queries.
-        with self.assertNumQueries(13):
```

Comment:
The discrepancy between this and the `self.assertNumQueries(15)` further down was due to this previously rendering the "no results" message rather than the table.